### ISSUE TITLE
Font text HD

### ISSUE THREAD

**Issue by Ihavedepressionimsad** [elishacloud/Silent-Hill-2-Enhancements#56]
Hi! Thanks to all the team for this project I loved!
(I don't speak English, sorry if you don't understand me)
edit*( I removed the text and the runtime error)
I see now a post with a user saying how to play the game even with the runtime error xD
Well, for now, the only issue I have but is not an issue I guess, for me, the font text doesn't look clear like HD text font ,I see on the video on youtube and in this screenshot below
http://enhanced.townofsilenthill.com/SH2/images/configComparisons/englishTextAdjustments/002.jpg
the text font really looks HD and very clear, but for me seems to be blurry, is there a way to make the text font change or fix it?
and again, thank you all the team for this project! I loved!

**Comment by Polymega** [comment]
Hi @Ihavedepressionimsad , 

I have an easy answer to why [this particular picture](http://enhanced.townofsilenthill.com/SH2/images/configComparisons/englishTextAdjustments/002.jpg)'s font looks so clean/sharp: I did it in Photoshop! 

For the Installation Guide website, I needed to include smaller pictures for the [Configuration Page](http://enhanced.townofsilenthill.com/SH2/config.htm#EnglishTextAdjustments) so as to visually show examples of the changes. The problem with that particular image was that the text was too small to read by default. So I replicated the font in Photoshop and made it much bigger so people would be able to see/read it on the small image on the website. I hope that made sense. 

We're not sure if "HD fonts" can be implemented into the game or not yet, but it may be something we'll try to look into sometime in the future.
